packages/headless/src/invoices.ts

```diff
@@ -15,6 +15,27 @@ import { MutationKey, QueryKey } from "@/constants";
 type OnSuccess = Void;
 type OnError = (error: Error) => void;
 
+type useFindInvoicesProps = {
+  query: UseInfiniteQueryResult<
+    InfiniteData<Paginated<IInvoice.Self>, unknown>,
+    Error
+  >;
+  list: IInvoice.Self[] | null;
+  more: () => void;
+};
+
+export function useFindInvoices({
+  pageParam,
+}: {
+  pageParam: number;
+}): useFindInvoicesProps {
```

Comment:
No need to pass the params this way. 
```suggestion
export function useFindInvoices(): useFindInvoicesProps {
```